CHANGELOG.md

```diff
@@ -42,17 +42,23 @@ All notable changes to this project will be documented in this file.
             * Methods added:
                 * `get_lcr_compensation_data`
     * #### Changed
+        * Added enum values to the `OutputStates` enum, and renamed existing enum values.
     * #### Removed
 * ### `nidigital` (NI-Digital Pattern Driver)
     * #### Added
     * #### Changed
     * #### Removed
 * ### `nidmm` (NI-DMM)
     * #### Added
+        * Added enum `DigitsResolution`, to clarify the available options for the `resolution_digits` attribute.
     * #### Changed
+        * The `resolution_digits` attribute's type is now `enums.DigitsResolution`.
```

Comment:
@marcoskirsch @ni-jfitzger This is a breaking change, because we require that the proper enum type is passed in; you can't just pass in a float that matches one of the enum values. Should I update the metadata to avoid it?

Reply:
I was about to bring this up. I don't think you should be making breaking changes.
i don't understand why you made this change in the first place.

Reply:
It was caused by a metadata update. In the source data, the enum has long been marked as 'codegen_method':'no', but a recent change moved it to a different layer, and I think I now need to explicitly add in the 'codegen_method':'no' at the python layer. I'll do that.

Reply:
we shouldn't introduce source breakers

Reply:
Ok, fixing in [!372493](https://ni.visualstudio.com/DevCentral/_git/ni-central/pullrequest/372493)

Reply:
Reverted this change.